_layouts/repo_instance.html

```diff
@@ -25,11 +24,23 @@
 {% for distro in page.all_distros %}
   <div class="distro distro-{{distro}}">
     <div class="container-fluid">
-      {% include repo_info.html %}
-      {% if page.available_distros[distro] or page.available_older_distros[distro] %}
-        {% assign repo = page.repo %}
+      {% unless page.available_distros[distro] or page.available_older_distros[distro] %}
+        {% for fallback_distro in page.all_distros %}
+          {% if page.available_distros[fallback_distro] or page.available_older_distros[fallback_distro] %}
+            {% assign rendered_distro = fallback_distro %}
+            {% break %}
```

Comment:
The effect of this break is that the fallback distro becomes the oldest available distro. Experimenting with a full build, this is usually humble. It seems possible to me that it might end up being the oldest distro in page.available_older_distros, but I could not find an example of this. Simply removing the break would probably not result in better performance, it might start using the available_older_distros as the result.

This would take some logic to fix. It's also kind of a nit, so probably should be a followup rather than changed now.

Reply:
```suggestion
            {% break %} {% comment %} tfoote: check ordering of fallback, to get newest. Maybe newest LTS if possible? {% endcomment %}
```

Thanks yeah that makes sense to keep an eye on.